.circleci/config.yml

```diff
@@ -11,9 +11,14 @@ jobs:
       - checkout
 
       - run:
-          name: Build WASM
-          command: bash build-wasm.sh WORMHOLE
-
+          name: Build WASM WORMHOLE
+          command: |
+            bash build-wasm.sh WORMHOLE
+            mkdir artifacts
+            cp build-wasm/bergamot-translator-worker.wasm artifacts/bergamot-translator-worker-with-wormhole.wasm
+            cp build-wasm/bergamot-translator-worker.js artifacts/bergamot-translator-worker-with-wormhole.js
+            shasum -a 256 ./artifacts/* > ./artifacts/SHA256-1
+            cp BERGAMOT_VERSION ./artifacts/
```

Comment:
@andrenatal I believe, the steps to copy the artifacts to "artifacts" folder should happen after the next step "Check artifacts" which checks whether artifacts are present or not.

Reply:
ok, fixed